First push of the ui-project

gunamata opened the pull request:
- Added the antora-ui-default project
- Added a helper file that filters our nav objects specific to the module of the current page

gunamata committed:
First commit including changes for language-switcher

Signed-off-by: Gunasekhar Matamalam <gunasekhar.matamalam@suse.com>

btat commented:
@gunamata Could you add the name(s) of the file(s) you've modified so we don't need to look at the files from upstream? It's not obvious since it's all one commit.

btat commented on .gitlab-ci.yml:
```diff
@@ -0,0 +1,55 @@
+image: node:10.14.2-stretch
```
This file should be removed.

gunamata replied:
Removed

btat commented on docs/antora.yml:
```diff
@@ -0,0 +1,5 @@
+name: antora-ui-default
```
We should remove the entire `docs` directory.

gunamata replied:
Removed

btat commented on .vscode/settings.json:
```diff
@@ -0,0 +1 @@
+{}
```
I think this got accidentally checked in and should be removed (and gitignored).

If there are plans for us to use shared settings, we should add this file when it has such settings.

gunamata replied:
Removed and added to .gitignore

btat replied:
This file is still showing up as of 6b1f8706beab9bfbc949ac6b144b0f21f47d378b.

gunamata commented:
> Could you add the name(s) of the file(s) you've modified so we don't need to look at the files from upstream? It's not obvious since it's all one commit.

Added file `src/helpers/navfilter.js`

gunamata committed:
Removed files that are not required

Signed-off-by: Gunasekhar Matamalam <gunasekhar.matamalam@suse.com>

gunamata committed:
Removed settings.json

Removed the LICENSE file now. To be added later after determining what's the correct license

Signed-off-by: Gunasekhar Matamalam <gunasekhar.matamalam@suse.com>
